### PR TITLE
Add nodejs install instructions

### PR DESCRIPTION
**Pull request by Sheila-nk**
I wanted to ensure the [setup instructions](https://pytest-jupyterhub.readthedocs.io/en/latest/contributing/setup.html) in the [contributing docs](https://pytest-jupyterhub.readthedocs.io/en/latest/contributing/index.html) work perfectly with no extra steps than stated. 
I realized tests were failing due to the missing `configurable-http-proxy` required by `JupyterHub`.
Added extra steps to install `nodejs`, `configurable-http-proxy` and `yarn`

### CODE REVIEW

**Review by consideRatio — APPROVED** [review]
LGTM!

If this was an older PR I'd press merge as part of approving it, but because its so recent I figure you should agree that its ready to be merged so I'll leave it to you to merge this if you think its ready!

**Comment by Sheila-nk** [comment]
> Btw, I'm not confident about yarn etc, is that required? Is yarn an alternative to npm?

I am not confident about it too. Seems like an alternative. I will exclude it from this project.

**Comment by Sheila-nk** [comment]
@consideRatio Thank you for reviewing this PR.

**Pull request merged**
